docs/installation.mdx

```diff
@@ -15,7 +15,11 @@ Choose the integration method suitable for your platform:
 
 ### HTML
 
-Add your website's CookieChimp JS snippet in the `<head>` tag of your HTML:
+Add your website's CookieChimp JS snippet at the top of the `<head>` tag of your HTML:
+
+<Info>
+  Add the CookieChimp script at the top of the `<head>` section to make sure it loads first.
```

Comment:
Say something like we need to set the consent default first so other scripts run accordingly. Eg. Google consent mode, etc. 

If we don't run first, other scripts may run and set cookies and other storage items when they should not be.